react/components/SearchResultContainer.js

```diff
@@ -39,8 +39,10 @@ export default class SearchResultContainer extends Component {
 
     const categoryWithChildrenReducer = (acc, category) => [
       ...acc,
-      category.Link,
-      ...category.Children.reduce(categoryReducer, []),
+      `/${category.Name}`,
+      ...category.Children.reduce(categoryReducer, []).map(
+        children => `/${category.Name}${children}`
```

Comment:
generally what should be the result of this expression? can you give an example?

Reply:
Yep. The original array would be something like:
```
[
  { 
    name: categoryName,
    ...,
    Children: [  { name: childrenCategoryName, ... }, ... ] 
  }
  ...
]
```
The final result would be: `['\categoryName', '\categoryName\childrenCategoryName']`.
This matches the shape that `productQuery` returns the categories (on `product-details`).

Reply:
@matheusps ah, I see

 I just thought this bit was weird: `/${category.Name}${children}`, because there are no separators between `${category.Name}` and `${children}`. Is this intentional?

Reply:
Ooops, my fault. It was correct, but the reduce was unnecessary. Just made that simpler 😅. Thanks